lib/Doctrine/Common/Reflection/StaticReflectionParser.php

```diff
@@ -153,8 +154,10 @@ protected function parse()
                     $docComment = $token[1];
                     break;
                 case T_CLASS:
-                    $this->docComment['class'] = $docComment;
-                    $docComment = '';
+                    if ($last_token !== T_PAAMAYIM_NEKUDOTAYIM) {
```

Comment:
Personally I think the constant is a good enough indicator on it's own. It's pretty easy to lookup if you are unfamiliar with it.